ni_measurementlink_service/_internal/stubs/ni/measurementlink/discovery/v1/discovery_service_pb2_grpc.pyi

```diff
@@ -4,19 +4,19 @@ isort:skip_file
 ---------------------------------------------------------------------
 ---------------------------------------------------------------------
 """
+
 import abc
 import collections.abc
 import grpc
 import grpc.aio
 import ni_measurementlink_service._internal.stubs.ni.measurementlink.discovery.v1.discovery_service_pb2 as ni_measurementlink_discovery_v1_discovery_service_pb2
 import typing
 
-_T = typing.TypeVar('_T')
+_T = typing.TypeVar("_T")
 
-class _MaybeAsyncIterator(collections.abc.AsyncIterator[_T], collections.abc.Iterator[_T], metaclass=abc.ABCMeta):
-    ...
+class _MaybeAsyncIterator(collections.abc.AsyncIterator[_T], collections.abc.Iterator[_T], metaclass=abc.ABCMeta): ...
 
-class _ServicerContext(grpc.ServicerContext, grpc.aio.ServicerContext):  # type: ignore
+class _ServicerContext(grpc.ServicerContext, grpc.aio.ServicerContext):  # type: ignore[misc, type-arg]
```

Comment:
Oops, I didn't look at the checks before I clicked "approve".

```
Run poetry run mypy -p ni_measurementlink_service
ni_measurementlink_service/_internal/stubs/session_pb2_grpc.pyi:17: error: Unused "type: ignore[type-arg]" comment  [unused-ignore]
ni_measurementlink_service/_internal/stubs/ni/protobuf/types/xydata_pb2_grpc.pyi:18: error: Unused "type: ignore[type-arg]" comment  [unused-ignore]
ni_measurementlink_service/_internal/stubs/ni/measurementlink/pin_map_context_pb2_grpc.pyi:16: error: Unused "type: ignore[type-arg]" comment  [unused-ignore]
ni_measurementlink_service/_internal/stubs/ni/measurementlink/discovery/v1/discovery_service_pb2_grpc.pyi:19: error: Unused "type: ignore[type-arg]" comment  [unused-ignore]
ni_measurementlink_service/_internal/stubs/ni/measurementlink/sessionmanagement/v1/session_management_service_pb2_grpc.pyi:[17](https://github.com/ni/measurementlink-python/actions/runs/8541294463/job/23400271232?pr=680#step:10:18): error: Unused "type: ignore[type-arg]" comment  [unused-ignore]
ni_measurementlink_service/_internal/stubs/ni/measurementlink/measurement/v2/measurement_service_pb2_grpc.pyi:19: error: Unused "type: ignore[type-arg]" comment  [unused-ignore]
ni_measurementlink_service/_internal/stubs/ni/measurementlink/measurement/v1/measurement_service_pb2_grpc.pyi:19: error: Unused "type: ignore[type-arg]" comment  [unused-ignore]
ni_measurementlink_service/_internal/stubs/ni/measurementlink/pinmap/v1/pin_map_service_pb2_grpc.pyi:17: error: Unused "type: ignore[type-arg]" comment  [unused-ignore]
Found 8 errors in 8 files (checked 85 source files)
Error: Process completed with exit code 1.
```

These are flagged by `warn_unused_ignores = true`.

The fix is to add something like this to pyproject.toml:
```
[[tool.mypy.overrides]]
# mypy-protobuf codegen has some unused ignores.
module = ["ni_measurementlink_service._internal.stubs.*"]
warn_unused_ignores = false
```